src/main/java/cms/gov/madie/measure/repositories/MeasurePatchRepositoryImpl.java

```diff
@@ -0,0 +1,49 @@
+package cms.gov.madie.measure.repositories;
+
+import cms.gov.madie.measure.exceptions.InternalServerException;
+import gov.cms.madie.models.measure.Measure;
+import org.springframework.data.mongodb.core.FindAndModifyOptions;
+import org.springframework.data.mongodb.core.MongoOperations;
+import org.springframework.data.mongodb.core.query.Update;
+import org.springframework.stereotype.Repository;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.springframework.data.mongodb.core.query.Criteria.where;
+import static org.springframework.data.mongodb.core.query.Query.query;
+
+@Repository
+public class MeasurePatchRepositoryImpl implements MeasurePatchRepository {
+  private final MongoOperations mongoOperations;
+
+  public MeasurePatchRepositoryImpl(MongoOperations mongoOperations) {
+    this.mongoOperations = mongoOperations;
+  }
+
+  @Override
+  public Measure findAndModify(Measure updatedMeasure) {
```

Comment:
would it be better to pass in the excludedFields list to this method so it's more generic?